packages/@ember/-internals/glimmer/tests/unit/runtime-resolver-cache-test.js

```diff
@@ -38,6 +39,8 @@ moduleFor(
           helperDefinitionCount: 1,
           // from this.render
           templateCacheMisses: 1,
+          // from debugRenderTree
+          templateCacheHits: ENV._DEBUG_RENDER_TREE ? 1 : 0,
```

Comment:
These values changed due to the new DebugRenderTree integration looking up templates an additional time